lib/wpcom-error-handler/wpcom-error-handler.php

```diff
@@ -53,9 +53,13 @@ function wpcom_get_error_backtrace( $last_error_file, $last_error_type, $for_irc
 			$path .= isset( $call['line'] ) ? ':' . $call['line'] : '';
 		}
 
-		if ( in_array( $call['function'], array( 'do_action', 'apply_filters' ) ) ) {
+		if ( isset( $call['class'] ) ) {
+			$path .= " {$call['class']}{$call['type']}{$call['function']}()";
```

Comment:
Nice.

Considering how many different combinations of params there might be, it might be worthwhile adding a check for type, or maybe preceding this line with
`$c_type = isset( $call['type'] ) ? $call['type'] : '???';`